app/controllers/application_controller.rb

```diff
@@ -27,6 +28,22 @@ def render(*args)
 
   private
 
+  # filter out restricted products, defined in flow coonsole
+  # https://console.flow.io/:organization/restrictions
+  def flow_filter_products
```

Comment:
I understand. that way I will get only localized items for particular experience.

so I should modify products search to show only products that have localization in current experience. if there is no localization info present, dont show it. if that is a bug, track why localization info is not present and do not fall back to solidus local prices.

right?